bin/foreman-installer

```diff
@@ -2,17 +2,8 @@
 require 'rubygems'
 require 'kafo'
 
-# base the answer file name on the file calling the
-# script.  This allows other programs to use their own
-# files if named based on their executable
-config_filename = File.basename($0) + ".yaml"
-
-# where to find answer file
-if File.exist?('config/' + config_filename)
-  CONFIG_FILE = 'config/' + config_filename
-else
-  CONFIG_FILE = '/etc/foreman/' + config_filename
-end
+# where to find scenarios
+CONFIG_DIR = '/etc/foreman/installer-scenarios.d/'
```

Comment:
The previous logic allowed running of the installer from a git checkout which is nice when testing changes to modules or the installer logic itself. Could we keep that?